### PR TITLE
add single process mint

### PR DESCRIPTION
**Pull request by brandonbryant12**

### CODE REVIEW

**Review by rafa-js — APPROVED** [review]
Same here. Waiting until the SDK is merged in case last minute changed pop up.

**Review by rafa-js — COMMENTED** [review]
Cool. One doubt.

File: src/loaders/HandCashItemsLoader.ts
Comment (brandonbryant12) [reply src/loaders/HandCashItemsLoader.ts:60]: yes - I will include in the next iteration because it will also include some breaking changes to some of the input types 

Same time we deprecate imageHighResUrl I think it makes sense

**Pull request merged**
